GettingToKnowMe.php

```diff
@@ -0,0 +1,61 @@
+<html>
+  <head>
+    <title> Getting to know me </title>
+    <link rel="stylesheet" href="style.css?Time=<?php echo microtime()?>" />
+  </head>
+<body>
+    <h1>
+        My Blog! </h1>
+        <div class = "active">
+              <ul>
+              <li>
+                  <a href = "index.php">
+                  Home
+              </a> </li>
+                  <li>
+                      <a href = "blogpost1.php">
+                          Contact
+                  </a> </li>
+
+                  <li>
+                      <a href = "Resume.php">
+                       Projects/Resume </a> </li>
+                <li>
+                    <a href = "blogpost2.php">
+                        Summer Blog </a> </li>
+              </ul>
+        </div>
+
+   <br>
+   <div style='max-width: 90%; background-color: #B3C3E4; padding:10px; margin:auto; border-radius:10px'>
```

Comment:
It looks like you repeat this line of code in each of your files (I'm guessing this is your main body wrapper). This is a perfect place to move the CSS out into an external file. Right now, if you want to make changes to your design, you have to edit every single file, but if you just give this div a class and then define a style for the class, you can change it everywhere at once.